docker-compose.yml

```diff
@@ -46,8 +46,8 @@ services:
       - postgres
     volumes:
       - log_volume:/dockstore_logs
-      - ./config/web.yml:/web.yml
-    command: ["dockerize", "-wait", "tcp://postgres:5432", "-timeout", "60s", "./init_migration.sh"]
+      - ./config/web.yml:/home/web.yml
```

Comment:
Did you confirm that the user has access to the /home dir?

Reply:
Yes, at least it works for me locally. The other pattern I saw was people putting the jar in /usr/app, but I don't see that in our container. I didn't want to also have to create the directory to limit the scope, so I looked around in the container while it was running and saw the /home directory